p2p/src/store.rs

```diff
@@ -127,6 +127,15 @@ impl PeerStore {
 		batch.commit()
 	}
 
+	pub fn save_peers(&self, p: Vec<PeerData>) -> Result<(), Error> {
```

Comment:
would it make sense to also redefine the `save_peer` above to delegate the call to `save_peers` for a vector of size 1?

Reply:
That would require every call to save_peer to create a Vec and add the single item.  Im not convinced thats better than having a few lines of similar code.

Reply:
I think the suggestion is to make save_peer just a wrapper around save_peers, instead of duplicating the batch DB code.